spec/controllers/users/events_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.describe Users::EventsController, type: :controller do
+  describe "GET #index" do
+    subject { get :index, user_id: user }
+
+    context "user has already signed in" do
+      login
+
+      context "user accesses the other one's events" do
+        let(:user) { create(:user) }
+
+        it_behaves_like "show error", ACCESS_DENIED do
+          before { subject }
+        end
+      end
+
+      context "user accesses his/her events" do
+        let(:user) { @user }
+        let(:event_1) { create(:event, creator: @user, created_at: 3.weeks.ago) }
+        let(:event_2) { create(:event, creator: @user, created_at: 2.weeks.ago) }
```

Comment:
I think it's an interesting fact that the user's Settings page is the only place (at the moment) which displays expired events, so let's create an expired event here to test the fact.